### PR TITLE
bump sdk version

### PR DESCRIPTION
**Pull request by nhynes**
re https://github.com/oasisprotocol/oasis-sdk/pull/77/

### CODE REVIEW

File: module-bridge/src/lib.rs [hunk 1]
```diff
@@ -210,6 +183,12 @@ lazy_static! {
     pub static ref ADDRESS_LOCKED_FUNDS: Address = Address::from_module(MODULE_NAME, "locked-funds");
 }
 
+macro_rules! id_to_key {
```
Comment (kostko) [inline module-bridge/src/lib.rs:186]: Why do you need a macro for this? It seems much clearer if you just use `.to_be_bytes()` inline.

Comment (nhynes) [reply module-bridge/src/lib.rs:186]: seemed kinda implementation details-y, but sure, I can inline it

**Review by pro-wh — APPROVED** [review]
fixes #29?

**Comment by kostko** [comment]
> fixes #29?

Yeah I already linked the issue above :-)

File: module-bridge/src/lib.rs (hunk 1 above)
Comment (kostko) [reply module-bridge/src/lib.rs]: That's a fair point, maybe a type would be cleaner than a macro though. Something like `IdentifierKey([u8; 8])` which you could create from an `u64` and would impl `AsRef<[u8]>`.

Comment (nhynes) [reply module-bridge/src/lib.rs]: I dig. Let me do that.

**Comment by nhynes** [comment]
@kostko are you opposed to a trait instead of single-purpose wrapper type? My personal opinion is that it's more readable, but I'll defer to your taste.

**Comment by kostko** [comment]
Yeah a trait would be great if you can make it not limited to the fixed-size `u64` use case.

**Comment by pro-wh** [comment]
> Yeah I already linked the issue above :-)

:laughing: thanks I didn't even realize you could do that other than by commenting

**Comment by nhynes** [comment]
> Yeah a trait would be great if you can make it not limited to the fixed-size u64 use case.

What other kind of bridge storage key is there? The whole point is just to write `id.to_storage_key()` instead of `to_storage_key(id)` or `StorageKey::from(id)`. That trait can't be upstreamed into `sdk` anyway because `u64` isn't necessarily always desired to be `be_bytes`ified.

If it weren't a trait, my next choice would be `fn to_storage_key(id: u64) -> [u8; 8]` since it's simple and fewer lines of code than `struct IdStorageKey([u8; 8])`

**Review by kostko — APPROVED** [review]

**Pull request merged**